.dockerignore

```diff
@@ -8,3 +8,4 @@ sdks
 vendor
 ui/dist
 ui/node_modules
+Dockerfile.windows
```

Comment:
Delete this - it is why the `docker-build` job is failing.

Reply:
removed